### PR TITLE
Gate assigner reboot with refresh

### PR DESCRIPTION
**Pull request by goulven04**
Re-enable the gate assigner in CoFrance:
- Refresh the gate every 30 seconds
- Add command `.cofrance stand` to disable/enable the auto assigner (enabled by default)
- Add command `.cofrance debug` to display the API queries and responses (disabled by default)

### CODE REVIEW

**Comment by Flox06** [comment]
@goulven04 we gave a try with Hervé on LFPG tonight, seems that NO query is sent to the server (according to server logs). We tried to toggle enable/disable using .cofrance stand, no changes.
Also enabled debug logs, we didn't see any log.
Need you help :)

**Comment by Flox06** [comment]
@goulven04 disregard, I forgot to enable LFPG in the remote config (FACEPALM)
Sorry :p

File: RadarScreen.cpp [hunk 1]
```diff
@@ -217,7 +217,7 @@ void RadarScreen::OnRefresh(HDC hDC, int Phase)
 				size_t tp_decimal_pos = tp_distanceText.find(".");
 				tp_distanceText = tp_distanceText.substr(0, tp_decimal_pos + 2) + "Nm";
 
-				tp_distanceText = tp_distanceText + " " + tp_headingText + "�";
+				tp_distanceText = tp_distanceText + " " + tp_headingText + "�";
```
Comment (pierr3) [inline RadarScreen.cpp]: Encoding issue here must be fixed.

File: RadarScreen.cpp [hunk 2]
```diff
@@ -268,7 +268,7 @@ void RadarScreen::OnRefresh(HDC hDC, int Phase)
 					size_t decimal_pos = distanceText.find(".");
 					distanceText = distanceText.substr(0, decimal_pos + 2)+"Nm";
 
-					distanceText = distanceText + " " + headingText + "�";
+					distanceText = distanceText + " " + headingText + "�";
```
Comment (pierr3) [inline RadarScreen.cpp]: Encoding issue here must be fixed

**Review by pierr3 — CHANGES_REQUESTED** [review]
@goulven04

File: RadarScreen.cpp (hunk 1 above)
Comment (goulven04) [reply RadarScreen.cpp]: Fixed

File: RadarScreen.cpp (hunk 2 above)
Comment (goulven04) [reply RadarScreen.cpp]: Fixed

**Pull request merged**